build/js/intlTelInput.js

```diff
@@ -218,6 +220,15 @@ https://github.com/Bluefieldscom/intl-tel-input.git
                 this.countryList.removeClass("v-hide").addClass("hide");
                 // this is useful in lots of places
                 this.countryListItems = this.countryList.children(".country");
+                // create dropdownContainer markup
+                if (this.options.dropdownContainer) {
+                    this.dropdown = $("<div>", {
+                        "class": "intl-tel-input iti-container"
+                    });
+                    $("<div>", {
+                        "class": "flag-dropdown"
+                    }).appendTo(this.dropdown).append(this.countryList);
```

Comment:
We've already appended this.countryList to this.flagsContainer earlier in this function - if we're appending it here then we shouldn't do that. We should put a condition around that earlier line so we don't do it twice! (see the line where we create this.countryList)